wordpress-mobile/gutenberg-mobile
# Dock the keyboard hide button to the right side of the screen

Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/350

This PR docks the keyboard hide button to the right side of the screen. Depends on https://github.com/WordPress/gutenberg/pull/13415

Android:
![keyboard1and](https://user-images.githubusercontent.com/5032900/51399096-2f869f80-1b56-11e9-9e3f-73977f7eae65.gif)

iOS:
![keyboard1ios](https://user-images.githubusercontent.com/5032900/51399154-4f1dc800-1b56-11e9-8bbb-30dfb6fc31d8.gif)

**To Test**

Testing prerequisites

For WPiOS

Checkout the PRs branch to any arbitrary folder and cd .. to it
run yarn install, yarn start
Open XCode WPiOS on the latest develop
Clean build folder on Xcode, and then run the app

For WPAndroid

open grade.properties at WordPress-Android folder
add wp.BUILD_GUTENBERG_FROM_SOURCE = true to grade.properties
checkout the PRs branch in the subrepo of WordPress-Android repo
cd to WordPress-Android/libs/gutenberg-mobile
run yarn install, yarn start
yarn wpandroid on a separate terminal in the same directory

Test steps

Check with different blocks and verify keyboard hide button is docked to the right side of the screen while other buttons can be scrolled horizontally
Verify also there's no UI anomaly due to https://github.com/wordpress-mobile/gutenberg-mobile/pull/480#issuecomment-455620000
